resources/pool.rb

```diff
@@ -21,19 +21,60 @@
 
 actions :add, :config, :delete, :start, :stop, :restart, :recycle
 
+# root
 attribute :pool_name, :kind_of => String, :name_attribute => true
-attribute :runtime_version, :kind_of => String
 attribute :no_managed_code, :kind_of => [TrueClass, FalseClass], :default => false
 attribute :pipeline_mode, :kind_of => Symbol, :equal_to => [:Integrated, :Classic]
-attribute :private_mem, :kind_of => Integer
-attribute :worker_idle_timeout, :kind_of => String
-attribute :recycle_after_time, :kind_of => String
-attribute :recycle_at_time, :kind_of => String
+attribute :runtime_version, :kind_of => String
+
+# add items
+attribute :start_mode, :kind_of => Symbol, :equal_to => [:AlwaysRunning, :OnDemand], :default => :OnDemand
+attribute :auto_start, :kind_of => [TrueClass, FalseClass], :default => true
+attribute :queue_length, :kind_of => Integer, :default => 1000
+attribute :thirty_two_bit, :kind_of => [TrueClass, FalseClass], :default => false
+
+# processModel items
 attribute :max_proc, :kind_of => Integer
-attribute :thirty_two_bit, :kind_of => Symbol
+attribute :load_user_profile, :kind_of => [TrueClass, FalseClass], :default => false
 attribute :pool_identity, :kind_of => Symbol, :equal_to => [:SpecificUser, :NetworkService, :LocalService, :LocalSystem, :ApplicationPoolIdentity ], :default => :ApplicationPoolIdentity
 attribute :pool_username, :kind_of => String
 attribute :pool_password, :kind_of => String
+attribute :set_profile_environment, :kind_of => [TrueClass, FalseClass], :default => true
+attribute :logon_type, :kind_of => Symbol, :equal_to => [:LogonBatch, :LogonService], :default => :LogonBatch
+attribute :manual_group_membership, :kind_of => [TrueClass, FalseClass], :default => false
+attribute :idle_timeout, :kind_of => String, :default => '00:20:00'
+attribute :shutdown_time_limit, :kind_of => String, :default => '00:01:30'
+attribute :startup_time_limit, :kind_of => String, :default => '00:01:30'
+attribute :pinging_enabled, :kind_of => [TrueClass, FalseClass], :default => true
+attribute :ping_interval, :kind_of => String, :default => '00:00:30'
```

Comment:
Sometimes you referred to this as pinging_interval instead of ping_interval as it's defined here.